demos/python_demos/action_recognition/action_recognition.py

```diff
@@ -27,11 +28,16 @@
 from action_recognition_demo.steps import run_pipeline
 from os import path
 
+sys.path.append(str(Path(__file__).resolve().parent.parent / 'common'))
```

Comment:
I'm `pathlib`'s biggest fan, but IMO, we shouldn't import it just for this. A total switch to `pathlib` would be nice, but if the rest of the demo's still using `os.path`, this should use it too.

Reply:
Updated